cernopendata/modules/fixtures/data/docs/delphi-guide-docker/delphi-guide-docker.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "body": {
+      "content": "delphi-guide-docker.md",
+      "format": "md"
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "short_description": {
+      "content": "This guide describes how to run the DELPHI stack from a docker container..."
```

Comment:
- Please use Docker with leading uppercase letter to match the case used in the title and the body.
- A simple trailing dot would be enough.

Reply:
Fixed. Thanks!